### PR TITLE
Add label smoothing in Semseg

### PR DESCRIPTION
**Pull request by thetianshuhuang**
Add a label_smoothing option to the Semseg objective, which is passed through to torch.nn.CrossEntropyLoss.

---

<sub>Stack created with <a href="https://github.com/github/gh-stack">GitHub Stacks CLI</a> • <a href="https://gh.io/stacks-feedback">Give Feedback 💬</a></sub>

### CODE REVIEW

**Review by huangjuite — APPROVED** [review]
LGTM!
